src/main/java/subway/line/LineService.java

```diff
@@ -22,29 +25,43 @@ public LineResponse saveLine(LineRequest request) {
 
         Line line = lineRepository.save(request.toEntity(upStation, downStation));
 
+        sectionRepository.save(new Section(line, upStation, downStation,
```

Comment:
line과 section이 연관관계 매핑이 되어 있어서 repository로 직접 지정을 안해줘도 될거에요.
line 객체에 section을 추가해주시고 line만 repository를 사용해서 저장하면 코드를 줄일 수 있겠네요.

Reply:
처음 라인이 저장될 때, 해당 라인은 구간에도 등록되어야하기 때문에 기존에는 직접 repository로 저장을 했습니다. 
현재는 `SectionRepository` 사용하지 않고, `Line`에 `addSection()`을 구현한 후 이를 사용하는 방향으로 개선했습니다..! 

```suggestion
        Section section = new Section(line, upStation, downStation, request.getDistance());
        line.addSection(section);
```

리뷰어님이 의도하신 방향과 맞게 개선되었을까요?